### ISSUE TITLE
Low F1 score on squad v1.1 even with low fine-tuning loss

### ISSUE THREAD

**Issue by kwanhong66** [google-research/bert#685]
I am doing pretraining and taking intermediate checkpoint to check downstream task performance which is squadv1.1

After fine-tuning done, loss for final step: 0.000116125324 which is 5 epochs and 18,877 steps.
It results in F1 score is 60 and em is 26.
I think fine-tuning loss is low enough to get good predict but it didn't

Is there any possibility of mistakes? or overfitting problem?
is it just needs more training steps for pretraining?
(pretraining run 250,000 steps and eval loss is 2.3)
